core/src/main/java/com/sequenceiq/cloudbreak/reactor/CollectDownscaleCandidatesHandler.java

```diff
@@ -52,8 +52,9 @@ public void accept(Event<CollectDownscaleCandidatesRequest> event) {
                 List<InstanceMetaData> notDeletedNodes = instanceMetaDataList.stream()
                         .filter(instanceMetaData -> !instanceMetaData.isTerminated() && !instanceMetaData.isDeletedOnProvider())
                         .collect(Collectors.toList());
-
-                ambariDecommissioner.verifyNodesAreRemovable(stack, notDeletedNodes);
+                if (!request.getDetails().isForced()) {
```

Comment:
Ambari will not finish decomission of nodes, if it violates replication rules. Should be refactored to STACK_DOWNSCALE.

Reply:
Decision was that, we'll leave out ambari decomission step if termination is forced.